nvim/nvim.go

```diff
@@ -187,6 +188,30 @@ func ChildProcessLogf(logf func(string, ...interface{})) ChildProcessOption {
 	}}
 }
 
+// ChildProcessDisableEmbed disables the --embed flag of nvim.
+// This is needed for applications to use the RPC API.
```

Comment:
Somewhat confusing. Why mention "this is needed for RPC" in a function that removes `--embed` ? 

```suggestion
// ChildProcessDisableEmbed disables the --embed flag of nvim.
```

Reply:
Yeah true, this one line should explain enough 👍